### ISSUE TITLE
Review Generic LotR Zones

### ISSUE THREAD

**Issue by datanoh** [trekcc-ops/gemp-stccg#3]
The Zone enum in Zone.java includes 6 classes from LotR that I'm not certain are well-integrated in the ST:CCG code or not:

- Support
- Attached
- Stacked
- Removed
- Void
- Void from Hand

Recommend looking into these classes and identifying if they are needed for ST:CCG. If so, recommend documentation to explain what they are and how they should be used consistently.

**Comment by datanoh** [comment]
Flagging @andrewd18 on this for his potential input on attached and stacked zones.

**Comment by datanoh** [comment]
Also worth noting - "attach" and "stack" are terms with specific meaning in the LotR card game. Neither are in our game. They're also not specific in Star Wars, but the Star Wars Gemp code has continued to use these zones.

I think it's a potential source of confusion as we get into more and more complicated rules and game text interactions, and it does keep us from having methods that are as dynamic as they could be. (We have a number of methods that assume zone == null means the card is in play as an awkward work around.)

**Comment by andrewd18** [comment]
I believe as of #172 zones are now purely a definition of the UI. They do probably still need to be reviewed and reworked; updating tags.

**Comment by datanoh** [comment]
Nope, we still have VOID and ATTACHED on the server side. I'm just no longer telling the client about them. Fine to split these out as separate issues if you want, but I'd like to still have a server issue for this.
